client/ayon_core/plugins/publish/integrate.py

```diff
@@ -924,8 +924,12 @@ def create_version_data(self, instance):
 
         # Include optional data if present in
         optionals = [
-            "frameStart", "frameEnd", "step",
-            "handleEnd", "handleStart", "sourceHashes"
+            "frameStart", "frameEnd",
+            "handleEnd", "handleStart",
+            "resolutionWidth", "resolutionHeight",
+            "pixelAspect",
+            "step",
+            "sourceHashes"
```

Comment:
Can we keep `step` near the frame and handles? Since it's frame related?
```suggestion
            "frameStart", "frameEnd",
            "handleEnd", "handleStart",
            "step",  # frame step size
            "resolutionWidth", "resolutionHeight",
            "pixelAspect",
            "sourceHashes"
```